radio/src/targets/common/arm/stm32/heartbeat_driver.cpp

```diff
@@ -79,6 +79,8 @@ void check_intmodule_heartbeat()
 #endif
     EXTI_ClearITPendingBit(INTMODULE_HEARTBEAT_EXTI_LINE);
 
+    mixerSchedulerResetTimer();
```

Comment:
@3djc I believe that reseting the timer here is good, thus allowing to do it earlier in the game, and removing the special casing for heartbeat in `setupPulsesInternalModule()`. It seems to me as well the timer reset was missing in the same function for PXX1 (`setupPulsesInternalModule()`):
```
#if defined(PXX1) && defined(INTMODULE_USART)
    case PROTOCOL_CHANNELS_PXX1_SERIAL:
      intmodulePulsesData.pxx_uart.setupFrame(INTERNAL_MODULE);
      return true;
#endif
```

Reply:
By the way, that would explain why the issue with iXJT showed up on X10/X12S, and not on Taranis-like targets, as only these targets use PXX1 and INTMODULE_USART.